### PR TITLE
Allow Prometheus to run on a tainted heapster node.

### PR DESCRIPTION
**Pull request by oxddr**
See https://github.com/kubernetes/kubernetes/issues/85683 and https://github.com/kubernetes/kubernetes/pull/85797 for context. 

/assign @wojtek-t

### CODE REVIEW

File: clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml [hunk 1]
```diff
@@ -34,6 +34,10 @@ spec:
     fsGroup: 2000
     runAsNonRoot: true
     runAsUser: 1000
+  tolerations:
+  - key: "monitoring"
+    operator: "Exists"
+    effect: "PreferNoSchedule"
```
Comment (wojtek-t) [inline clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml]: Prefer won't force anything - is that really what you want?
Shouldn't we use `NoSchedule`

Comment (oxddr) [reply clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml]: Yes, I don't think it's going to be a problem if we schedule one or two hollow nodes or other infrastructure pods on this node. I don't have a strong preference on this though.

Comment (oxddr) [reply clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml]: Done

**Pull request merged**